packages/mdc-button/README.md

```diff
@@ -40,38 +40,54 @@ npm install --save @material/button
 
 ## Usage
 
-### Flat
+### Button type
 
-```html
-<button class="mdc-button">
-  Flat button
-</button>
-```
+> note: Examples and documents use generic `<button>`, but users can also apply
```

Comment:
I actually think we use this pattern in README note in general.
See `mdc-typograph` and `mdc-card`.